variables.tf

```diff
@@ -13,6 +13,12 @@ variable "allowed_account_ids_write" {
   type        = list(string)
 }
 
+variable "dest_region" {
+  default     = ""
```

Comment:
`default = null` probably better here

Reply:
What benefit does that give?

Reply:
terraform has lots of syntax these days for whether a var was intentionally omitted. `try()` is an example of something that works nicely when null is used but does not work when an empty str is used instead.